libc/src/sys/socket/linux/recvmsg.cpp

```diff
@@ -40,9 +40,11 @@ LLVM_LIBC_FUNCTION(ssize_t, recvmsg,
   }
 
   // Unpoison the msghdr, as well as all its components.
+  MSAN_UNPOISON(msg, sizeof(struct msghdr));
```

Comment:
nit: is there any name conflict with `msghdr`?  If not just use `msghdr` instead of `struct msghdr` in this file.

Reply:
Done.